rendermodules/rough_align/apply_rough_alignment_to_montages.py

```diff
@@ -68,6 +69,7 @@ def apply_rough_alignment(render,
                           output_dir,
                           scale,
                           Z,
+                          apply_scale=False,
                           consolidateTransforms=True):
```

Comment:
I don't think apply_scale is in the schema for this and it is never passed to this function.